lib/seattleflu/id3c/cli/command/etl/redcap_det_fh_airs.py

```diff
@@ -0,0 +1,910 @@
+"""
+Process DETs for the Fred Hutch AIRS study into the FHIR receiving table
+"""
+import logging
+from dateutil.relativedelta import relativedelta
+from enum import Enum
+from id3c.cli.command.etl import redcap_det
+from id3c.cli.redcap import is_complete, Record as REDCapRecord
+from id3c.db import find_identifier
+from seattleflu.id3c.cli.command import age_ceiling
+from .redcap import *
+from .redcap_map import *
+
+LOG = logging.getLogger(__name__)
+
+
+class AIRSProject():
+    id: int
+    lang: str
+    command_name: str
+
+    def __init__(self, project_id: int, lang: str, command_name: str) -> None:
+        self.id = project_id
+        self.lang = lang
+        self.command_name = command_name
+
+
+REDCAP_URL  = 'https://redcap.fredhutch.org/'     # REDCap server hostname
+REDCAP_PID  = 1372                                # REDCap project ID
+REDCAP_LANG = 'en'                                # Survey language
+ETL_COMMAND = 'fh-airs'                           # Run as 'id3c etl redcap-det <ETL_COMMAND>'
+
+PROJECTS = [
+        AIRSProject(REDCAP_PID, REDCAP_LANG, ETL_COMMAND)
+    ]
+
+LANGUAGE_CODE = {
+project.id: project.lang
+    for project in PROJECTS }
+
+class CollectionMethod(Enum):
+    SWAB_AND_SEND = 'swab_and_send'
+    KIOSK = 'kiosk'
+    UW_DROPBOX = 'uw_dropbox'
```

Comment:
I just confirmed with Cooper that this study is swab and send only, so you should be safe to drop out all the logic for drop box and kiosk and the TODO a few lines down.

Reply:
Noted and removed.